draft-opsarea-rfc5706bis.md

```diff
@@ -1153,6 +1162,64 @@ Information Models and Data Models
    Protocol designers should consider both data-oriented and task-
    oriented authority levels and policy.
 
+## Tooling Considerations {#sec-tooling}
+
+   The operational community's ability to effectively adopt and
+   use new specifications is significantly influenced by the availability
+   and adaptability of appropriate tooling. While the introduction of a
+   new specification does not automatically mandate the development of entirely
+   new tools, careful consideration must be given to how existing tools can
+   be leveraged or extended to support the management and operation of these
+   new specifications.
+
+   The {{NEMOPS}} workshop highlighted a consistent theme applicable beyond
+   network management protocols: the "ease of use" and adaptability of existing
+   tools are critical factors for successful adoption. Therefore, a new
+   specification should provide examples using existing, common tooling,
+   or running code that demonstrate how to perform key operational tasks.
+
+   Specifically, the following tooling-related aspects should be considered,
+   prioritizing the adaptation of existing tools:
+
+   *  Leveraging Existing Tooling: Before considering new tools, assess whether
+      existing tooling, such as monitoring systems, logging platforms,
+      configuration management systems, and orchestration frameworks, can be
+      adapted to support the new specification. This may involve developing
+      plugins, modules, or drivers that enable these tools to interact with with
+      the new specification.
+
+  *  Extending Existing Tools: Identify areas where existing tools can be
+     extended to provide the necessary visibility and control over the new
+     specification. For example, if a new transport protocol is introduced,
+     consider whether existing network monitoring tools can be extended to
+     track its performance metrics or whether existing security tools can be
+     adapted to analyze its traffic patterns.
+
+  *  New Tools as a Last Resort: Only when existing tools are demonstrably
+     inadequate for managing and operating the new protocol should the development
+     of new tools be considered. In such cases, carefully define the specific
+     requirements for these new tools, focusing on the functionalities that cannot
+     be achieved through adaptation or extension of existing solutions.
+
+  *  IETF {{HACKATHONS}} for Manageability Testing: IETF {{HACKATHONS}}, held before each
+     IETF meeting, provide an opportunity to test the functionality,
```

Comment:
```suggestion
     provide an opportunity to test the functionality,
```

May not age well if we decide in the future to hold hackathon following another time plan (including seperate from week meetings)